### ISSUE TITLE
Add support for a new rates endpoint in limes

### ISSUE THREAD

**Issue by kayrus** [sapcc/limesctl#100]
Add support for https://github.com/sapcc/limes/pull/163

```
$ limesctl project show-rates project --services email-aws 
limesctl: error: could not get project report: Bad request with: [GET https://limes:8080/v1/domains/domain/projects/project?rates=only&service=svc], error message: query parameter "rates" is not supported anymore
```

**Comment by talal** [comment]
I'll have a look when I'm back in office.

**Comment by talal** [comment]
Fixed in v3.0.3
